tests/unit/gensym_test.py

```diff
@@ -0,0 +1,40 @@
+import pytest 
+
+from lvm_gensym import * 
+
+used_vg_names = ['rhel_user', 'rhel_user_0', 'rhel_user_1']
+used_lv_names = ['root', 'root_0', 'root_1', 'root_2', 'swap_0', 'swap', 'swap_1']
+test_input = [ ['ext', '/'], ['swap', '/'], ['ext', '/'], ['ext', '']]
+vg_names = ['rhel_user_2', 'rhel_user_3', 'rhel_user_4']
+
+def test_return_val():
+	index = 0
+
+	for names in vg_names:
+		assert is_unique(vg_names[index], used_vg_names) == True 
+		index += 1 
+
+def test_init_vg_name():
+	assert init_vg_name('hostname', vg_names) == 'rhel_hostname'
+	assert init_vg_name('user', vg_names) == 'rhel_user'
+
+def test_vg_eval():
+	assert get_vg_name('user', used_vg_names) == 'rhel_user_2'
+	assert get_vg_name('user', used_vg_names) == 'rhel_user_3'
+	assert get_vg_name('user', used_vg_names) == 'rhel_user_4'
+	assert get_vg_name('new_user', used_vg_names) == 'rhel_new_user'
+	assert get_vg_name('1234asdbc', used_vg_names) == 'rhel_1234asdbc'
+	assert get_vg_name('', used_vg_names) == 'rhel_default'
+	assert get_vg_name(None, used_vg_names) == 'rhel_default_0'
+
+def test_init_lv_name():
+	assert init_lv_name(test_input[0], used_lv_names) == 'root'
+	assert init_lv_name(test_input[1], used_lv_names) == 'swap' 
+	assert init_lv_name(test_input[3], used_lv_names) == 'mount_default' 
+
+def test_lv_eval():
+	counter = 0
+	for item in test_input:
+		expected = ['root_3', 'swap_2', 'root_4', 'mount_default']
```

Comment:
You should move the definition of `expected` outside of the loop since it doesn't change as you iterate over `test_input`. This is what's known as a loop invariant.